plugins/broker-redis/src/static/generateRedisClientOptions.ts

```diff
@@ -38,13 +37,19 @@ export const generateRedisClientOptions = (
     );
   }
 
-  return {
+  const redisOptions: RedisOptions = {
     transport: Transport.REDIS,
     options: {
-      url: redisUrl,
+      host: redisHost,
+      port: redisPort,
       retryAttempts: redisRetryAttempts,
       retryDelay: redisRetryDelay,
-      tls: redisEnableTLS,
     },
   };
+
+  if (redisEnableTLS) {
+    redisOptions.options!.tls = {};
+  }
```

Comment:
Can we really be sure that we do have `redisOptions.options`
why not to do something like that
```suggestion
  if (redisEnableTLS && redisOptions.options) {
    redisOptions.options.tls = {};
  }
```

Reply:
good one! Committing